main.java

```diff
@@ -0,0 +1,3 @@
+public void main(String args[]){
+System.out.println("This is main programme");
```

Comment:
This is the way to print. Well done! Your future at Ricoh is ready!

Reply:
Thank you for the review. Actually it was very difficult code,

Reply:
:-)

Reply:
🥇